feature-utils/poly-look/rollup.config.js

```diff
@@ -16,7 +16,7 @@ export default {
     },
   },
   plugins: [
-    svg(),
+    svg({ base64: true }),
```

Comment:
OK, I see that this fixes the fact that it was not showing the button; however, that had been there for a long time, so that can't be the cause it was not showing the button, right? Or it had not worked from the get go?

Reply:
Honestly I was surprised that the icon wasn't showing. When I did the IconButton, my mistake was that I've only checked Storybook and the code from dist. Since I saw something like `var x="<svg.....` I assumed it would just work. I didn't wanted to deal with copying assets and making paths work, so I took a closer look at that plugin, saw that option and tried it out.

Reply:
If it works, well, it works; however if it stopped working at some time in the past, it might have been due to some other reason, maybe change in CSS or who knows what. It's going to be almost impossible to know that, anyhow, and the person who added that is not here anymore, so I guess we're good with that.